src/components/ctf-components/ctf-text-banner/ctf-text-banner-ggl.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
```

Comment:
what is this `-ggl` suffix?

Reply:
I think it's used internal for retrieving the initial data before it's cached (not sure though), it's also used in the contentful examples so I sticked with it

Reply:
You mean `-gql` suffix then?